src/Kernel32.Shared/Win32Exception.cs

```diff
@@ -46,7 +46,7 @@ public Win32Exception()
         /// <param name="error">The Win32 error code associated with this exception.</param>
         public Win32Exception(Win32ErrorCode error)
 #if DESKTOP
-            : base((int)error)
+            : this(error, error.GetMessage() ?? $"Unknown error (0x{(int)error:x8})")
```

Comment:
If we were to keep this change, we might as well drop the `#if` that surrounds it since both branches are the same.